### DISCUSSION TITLE
wrong link for hawkeye

### DISCUSSION THREAD

**Discussion by pramodkvarma** [Sunbird-Obsrv/Community#6] [Issues]
@rhwarrier page https://obsrv.sunbird.org/learn/product-and-developer-guide/report-configurator points to hawkeye PDF with DIKSHA logo etc. You need to move that to a proper Obsrv doc. Please track and fix.

**Comment by rhwarrier** [comment]
Will update to an SB doc, thanks.

**Comment by rhwarrier** [comment]
The document has been updated.
